actors/sunbird-lms-mw/actors/sunbird-utils/sunbird-platform-core/common-util/src/main/resources/externalresource.properties

```diff
@@ -162,3 +162,4 @@ adminutil_sign_endpoint = v1/sign/payload
 self_declared_mandatory_fields = Diksha UUID,Status,State provided ext. ID,Channel,Persona
 self_declared_optional_fields = School Name,School UDISE ID,Email ID,Phone number,Error Type
 enable_captcha=true
+consent_expiry_in_days=3
```

Comment:
Would expect this to be made available in hours instead. May be ok for now. Let @maheshkumargangula advise.

Reply:
updated to 100 days as per mahesh's suggestion